Rakefile

```diff
@@ -0,0 +1,114 @@
+require 'json'
+require 'faraday'
+require 'fileutils'
+require 'logger'
+require 'rubygems'
+
+def logger
+  @logger ||= Logger.new STDOUT
+end
+
+def files
+  @files ||= []
+end
+
+task :default => [:update_static_files]
+
+task 'assets:precompile' => :update_static_files
+
+def version_for(version_str)
+  Gem::Version.new version_str.match(/\d+(\.\d+)*/)[0]
+end
+
+def fetch_githubusercontent_file(from, to = nil)
+  conn = Faraday.new('https://raw.githubusercontent.com')
+  public_files_dir = "public/files"
+  to = File.basename(from) unless to
+  to = File.join("..", public_files_dir, to)
+
+  FileUtils.mkdir_p public_files_dir
+
+  response = conn.get do |req|
+    logger.info "Fetching #{conn.url_prefix.to_s}#{from}"
+    req.url from
+  end
+
+
+  dest = File.expand_path(to, __FILE__)
+
+  if response.success?
+    logger.info "Writing to #{dest}"
+    File.write(dest, response.body)
+    logger.info "Setting mode 'a+rx' on #{dest}"
+    FileUtils.chmod "a+rx", dest
+    files << dest
+  else
+    fail "Could not fetch #{from}"
+  end
+end
+
+
+def latest_release_for(repo)
+  conn = Faraday.new('https://api.github.com')
+
+  response = conn.get do |req|
+    releases_url = "repos/#{repo}/releases"
+    logger.info "Fetching releases from #{conn.url_prefix.to_s}#{releases_url}"
+    req.url releases_url
+    oauth_token = ENV['GITHUB_OAUTH_TOKEN']
+    if oauth_token
+      req.headers['Authorization'] = "token #{oauth_token}"
+    end
```

Comment:
On Heroku, it is less of a concern to have `GITHUB_OAUTH_TOKEN` defined, since the rate limit is less likely to apply.